parm/soca/berror/soca_parameters_diffusion_vt.yaml

```diff
@@ -0,0 +1,30 @@
+geometry: &geom
+  mom6_input_nml: mom_input.nml
+  fields metadata: ./fields_metadata.yaml
+
+background:
+  read_from_file: 1
+  basename: ./INPUT/
+  ocn_filename: MOM.res.nc
+  ice_filename: cice.res.nc
+  date: '{{ATM_WINDOW_BEGIN}}'
+  state variables: [cicen, hicen, hsnon, socn, tocn, uocn, vocn, ssh]
+
+background error:
+  covariance model: SABER
+  saber central block:
+    saber block name: EXPLICIT_DIFFUSION
+    geometry: *geom
+    calibration:
+      normalization:
+        # NOTE, not actually used here, since the normalization spec is only used for hz
+        method: randomization  #< other option is "brute force"
+        iterations: 1000       #< in the real world you'll want to use 1e4 or so
+
+      groups:
+      - name: vt_ocean
+        vertical:
+          as gaussian: true
+          fixed value: 15.0
```

Comment:
this is the number of levels, btw. So a bit deep.

Reply:
Ha! Thanks. I was wondering ...